docs/src/pages/de/migration/0.21.0.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Umstellung auf v0.21
+description: Projekte von Astro v0.20 umziehen.
+---
+
+## Vite
+
+Beginnend mit v0.21 werden Astro-Builds mittels [Vite](https://vitejs.dev) kompiliert.
+Bedingt hierdurch sollten Konfigurationen, die in `snowpack.config.mjs` geschrieben wurden, nach `astro.config.mjs` transferiert werden.
+
+```js
+// @ts-check
+
+/** @type {import('astro').AstroUserConfig} */
+export default {
+  renderers: [],
+  vite: {
+    plugins: [],
+  },
+};
+```
+
+Um mehr über die Konfiguration von Vite zu erfahren, lies bitte ihren [Configuration Guide](https://vitejs.dev/config).
+
+## Aliasierung
+
+In Astro v0.21 können Aliase in `tsconfig.json` oder `jsconfig.json` hinzugefügt werden, um sie bei der Verwendung von _import_ einzusetzen.
+
+```json
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "@/components/*": ["components/*"]
+    }
+  }
+}
+```
+
+_Diese Aliase werden automatisch in die Konfiguration von [VSCode](https://code.visualstudio.com/docs/languages/jsconfig) und anderen Editoren übernommen._
+
+## Variablen in \<script> & \<style>
+
+In Astro v0.21 können _serialisierbare_ Server-seitige Variablen innerhalb Client-seitiger `<style>`- oder `<script>`-Tags übergeben werden.
+
+```astro
+---
+// tick.astro
+const foregroundColor = "rgb(221 243 228)";
+const backgroundColor = "rgb(24 121 78)";
+---
+<style define:vars={{foregroundColor, backgroundColor}}>
+  h-tick {
+    background-color: var(--backgroundColor);
+    border-radius: 50%;
+    color: var(--foregroundColor);
+    height: 15px;
+    width: 15px;
+  }
+</style>
+<h-tick>✓</h-tick>
+```
+
+## Komponenten in Markdown
+
+In Astro v0.21 können Komponenten aus beliebigen Frameworks in Markdown-Dateien verwendet werden.
+
+```markdown
+---
+Layout: '...'
+setup: |
+  import MeineReactKomponente from '../components/MeineReactKomponente.jsx'
+---
+
+# Anreicherung bei Sichtbarkeit
+
+<MeineReactKomponente client:visible>
+
+# Hallo Welt!
+
+</MeineReactKomponente>
+```
+
+## Komponenten in Titeln(Frontmatter)
+
+Bisher ließen sich Astro-Komponenten innerhalb des Astro-Frontmatter erzeugen, indem dort JSX-Syntax anstelle der Astro-Syntax verwendet wurde. Dieses Verfahren hatte stets etwas von einem Hack, und dieser kann mit dem neuen Kompilierer nicht weiter unterstützt werden. Wir hoffen, dass wir diese Möglichkeit in einer zukünftigen Version von Astro wieder einführen können - in der wir hierfür jedoch eine andere API, an Stelle der JSX-API, verwenden werden.
+
+Um auf v0.21 umzustellen, konvertiere bitte alle JSX-Astro-Komponenten (dies sind jedwede Astro-Komponenten, die innerhalb des Frontmatter einer anderen Komponente erzeugt werden) in eigenständige Komponenten.
```

Comment:
```suggestion
Um auf v0.21 umzustellen, erstelle bitte eigene Komponenten für alle JSX-Astro-Komponenten (dies sind Astro-Komponenten, die innerhalb des Frontmatter einer anderen Komponente erzeugt werden).

Reply:
I think it is clearly about already existing components to be converted, not about building new components. So nope. But I omitted 'jedwede')))